### PR TITLE
Bazel-iOS Con 2023: announce/ open call for presentations

### PR DESCRIPTION
**Pull request by jerrymarino**

### CODE REVIEW

File: docs/2023_bazel_ios_con.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Bazel-iOS Con 2023: open call for presentations - RSVPs
+
+Congratulations everyone 🎉!  I formally invite you to join us for a day of open
+community and fun, and low level iOS build talks. We'll additionally host a
+number of breakout rooms including: `rules_ios`, new-around-here, and at-scale
+building.
+
+The event will be held on September 27th at $TBD, Oakland, CA from 10:00AM -
+6:00PST. Registration and social hour at 9:00AM. We encorge folks to come IRL
```
Comment (justinseanmartin) [inline docs/2023_bazel_ios_con.md:9]:
```suggestion
6:00PST. Registration and social hour at 9:00AM. We encourage folks to come IRL
```

**Review by justinseanmartin — APPROVED** [review]

**Comment by ob** [comment]
Shouldn't this go in https://github.com/bazel-ios/bazel-ios.github.io?

**Review by thiagohmcruz — APPROVED** [review]

**Comment by jerrymarino** [comment]
Sure we can link it up there. For now, I've done the same as last year - just put adjacent to the other one inside of `docs`. Also happy to move stuff around or do a followup, and of course the event in general 🚀

**Comment by mattrobmattrob** [comment]
I guess this didn't happen, @jerrymarino? Is it happening later in the year instead?

**Comment by mattrobmattrob** [comment]
Closing since it's now 2024. 😆 I missed seeing you all but hopefully we can do something this year.

**Pull request closed without merging**